lldb/test/API/commands/command/script/exception/TestCommandException.py

```diff
@@ -0,0 +1,27 @@
+import os
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+
+
+class TestCase(TestBase):
+    NO_DEBUG_INFO_TESTCASE = True
+
+    def test(self):
+        """
+        Check that a Python command, which raises an unhandled exception, has
+        its status set to failed.
+        """
+        command_path = os.path.join(self.getSourceDir(), "throw_command.py")
+        self.runCmd(f"command script import {command_path}")
+
+        with open(os.devnull, "w") as devnull:
+            self.dbg.SetErrorFileHandle(devnull, False)
```

Comment:
Should the context manager reset `self.dbg`'s error file handle after this or is `self.dbg` remade for every test?

Reply:
A few tests do reset do reset the file handle (however some tests do not). I'll add a reset here too.